docs/editors/helix.md

```diff
@@ -0,0 +1,26 @@
+---
+id: helix
+sidebar_label: Helix
+title: Helix
+---
+
+![helix demo](https://i.imgur.com/b0sETIY.gif)
+
```

Comment:
If you want you can also add in a 
``````
```scala mdoc:requirements
```
``````

Which will embed the requirements for Metals (JDK, supported versions, etc) like the other editor pages have.